src/Blazor.EventGridViewer.Core/Blazor.EventGridViewer.Core.csproj

```diff
@@ -5,7 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.10.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="System.Memory.Data" Version="6.0.0" />
```

Comment:
This package will get pulled in by Azure.Messaging.EventGrid. Unless we need 6.0.0, I'd just remove this.